src/components/Section.tsx

```diff
@@ -22,41 +21,41 @@ export const Section = ({ onValidation }: {
   const selectedOptions = (allSections || [])?.filter(s => s.value === section?.uuid)
 
   return (
-    <Awareness name='Section' ref={setFocused} className='flex flex-col gap-2'>
-      <ComboBox
-        max={1}
-        size='xs'
-        modal={true}
-        sortOrder='label'
-        options={allSections}
-        selectedOptions={selectedOptions}
-        placeholder={section?.title || 'Lägg till sektion'}
-        onOpenChange={(isOpen: boolean) => {
-          if (setFocused?.current) {
-            setFocused.current(isOpen)
-          }
-        }}
-        onSelect={(option) => {
-          if (section?.title === option.label) {
-            setSection(undefined)
-          } else {
-            setSection(Block.create({
-              type: 'core/section',
-              rel: 'section',
-              uuid: option.value,
-              title: option.label
-            }))
-          }
-        }}
-      />
-      {onValidation &&
-        <Validation
-          label='Sektion'
-          path={path}
-          block='core/section[0]'
-          onValidation={onValidation}
+    <Awareness name='Section' ref={setFocused}>
+      <Validation
+        label='Sektion'
+        path={path}
+        block='core/section[0]'
+        onValidation={onValidation}
+      >
+        <ComboBox
+          max={1}
+          size='xs'
+          modal={true}
+          sortOrder='label'
+          options={allSections}
+          selectedOptions={selectedOptions}
+          placeholder={section?.title || 'Lägg till sektion'}
+          validation={!!onValidation}
```

Comment:
`Property 'validation' does not exist on type 'IntrinsicAttributes & ComboBoxControlledProps'.`

Reply:
Get the latest, install latest @ttab/elephant-ui and it should work :crossed_fingers: